real_check/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+add_subdirectory(test)
```

Comment:
Nitpick: consider adding a new line before add_subdirectory for readability.

Reply:
Done.